include/micm/solver/rosenbrock.hpp

```diff
@@ -246,17 +259,23 @@ namespace micm
     std::size_t n_params = 0;
     for (const auto& process : processes_)
     {
-      n_params += process.rate_constant_->SizeCustomParameters();
+      if (process.rate_constant_) n_params += process.rate_constant_->SizeCustomParameters();
     }
-    return State<>{ micm::StateParameters{ .state_variable_names_ = system_.UniqueNames(),
-                                         .number_of_grid_cells_ = parameters_.number_of_grid_cells_,
-                                         .number_of_custom_parameters_ = n_params,
-                                         .number_of_rate_constants_ = processes_.size() } };
+    return State<MatrixPolicy>{ micm::StateParameters{ .state_variable_names_ = system_.UniqueNames(),
+                                                       .number_of_grid_cells_ = parameters_.number_of_grid_cells_,
+                                                       .number_of_custom_parameters_ = n_params,
+                                                       .number_of_rate_constants_ = processes_.size() } };
   }
 
   template<template<class> class MatrixPolicy, template<class> class SparseMatrixPolicy>
-  inline Solver::SolverResult RosenbrockSolver<MatrixPolicy, SparseMatrixPolicy>::Solve(double time_start, double time_end, State<MatrixPolicy>& state) noexcept
+  inline Solver::SolverResult RosenbrockSolver<MatrixPolicy, SparseMatrixPolicy>::Solve(
+      double time_start,
+      double time_end,
+      State<MatrixPolicy>& state) noexcept
   {
+    Solver::SolverResult result{};
+    stats_.reset();
+#if 0
```

Comment:
is this here on purpose?

Reply:
Yes, this function needs to be updated, but it's part of a separate issue (#112)